integration_tests/tests/integration_tests/batch_mint_tests.rs

```diff
@@ -552,7 +553,15 @@ async fn batch_mint_with_unverified_creators_test() {
     // generate batch mint with creators verified value set to true
     // but signatures will not be attached
     // batch should not be saved
-    let test_batch_mint = generate_batch_mint(10, true);
+    let mut test_batch_mint = generate_batch_mint(10);
+
+    // set creators verified to true for this test case
+    for b_mint in test_batch_mint.batch_mints.iter_mut() {
```

Comment:
we can bring creator_verified flag setup to SDK generate_batch_mint() fn side. seems like it is not affected tree's hashes, but it may be more convenient

Reply:
Also had such an idea but rejected it because we need verified creators only in one test but changing SDK will also require changes in other repos which use this method. So it's +2 PRs for small change, thats why decided to do it this way 🙂